docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -233,151 +252,152 @@ kubectl --namespace quick-start-backend-ns  apply -f pg-service.yml
-4. Store the application user's credentials (held in the environment variables `APPLICATION_DB_USER` and `APPLICATION_DB_INITIAL_PASSWORD`) in in a secret store (for the purposes of this demo, we're using Kubernetes secrets).
+Next, we'll create the application database and user, and securely store the
+user's credentials:
+
+1. Create the application database
+1. Create the `pets` table in that database
+1. Create an application user with limited privileges: `SELECT` and `INSERT` on
+   the `pets` table
+1. Store these database **application-credentials** in Kubernetes secrets.
+
+So we can refer to them later, export the database name and
+application-credentials as environment variables:
 
-**Note:** You must set the value of and export the environment variables `APPLICATION_DB_NAME`, `APPLICATION_DB_USER` and `APPLICATION_DB_INITIAL_PASSWORD` before proceeding, e.g.
 ``` bash
 export APPLICATION_DB_NAME=quick_start_db
 export APPLICATION_DB_USER=app_user
 export APPLICATION_DB_INITIAL_PASSWORD=app_user_password
 ```
 
-To create the application database, application table, application user and grant the application user relevant privileges, run this command:
+Finally, to perform the 4 steps listed above, run:
 
 ```bash
-docker run \
-  --rm \
-  -i \
-  -e PGPASSWORD=${REMOTE_DB_ADMIN_PASSWORD} \
-  postgres:9.6 \
-    psql -U ${REMOTE_DB_ADMIN_USER} "postgres://${REMOTE_DB_URL}/postgres" \
+docker run --rm -i -e PGPASSWORD=${SECURITY_ADMIN_PASSWORD} postgres:9.6 \
+    psql -U ${SECURITY_ADMIN_USER} "postgres://${REMOTE_DB_URL}/postgres" \
     << EOSQL
-/* Create Application Database */
+
 CREATE DATABASE ${APPLICATION_DB_NAME};
 
-/* Connect to Application Database */
-\c ${APPLICATION_DB_NAME};
+\c ${APPLICATION_DB_NAME}; -- connect to it
 
-/* Create Application Table */
 CREATE TABLE pets (
   id serial primary key,
   name varchar(256)
 );
 
-/* Create Application User */
 CREATE USER ${APPLICATION_DB_USER} PASSWORD '${APPLICATION_DB_INITIAL_PASSWORD}';
 
 /* Grant Permissions */
+
 GRANT SELECT, INSERT ON public.pets TO ${APPLICATION_DB_USER};
 GRANT USAGE, SELECT ON SEQUENCE public.pets_id_seq TO ${APPLICATION_DB_USER};
 EOSQL
 ```
 <pre>
 CREATE DATABASE
-You are now connected to database "quick_start_db" as user "quick_start_admin_user".
+You are now connected to database "quick_start_db" as user "security_admin_user".
 CREATE TABLE
 CREATE ROLE
 GRANT
 GRANT
 </pre>
 
-### Create Application Namespace and Store Application DB-Credentials
-
-Now that the storage backend is setup and good to go, it's time to set up the namespace where the application will be deployed.
+### Create Application Namespace and Store Credentials
 
 The application will be scoped to the **quick-start-application-ns** namespace.
 
-Run this code to create the namespace:
+To create the namespace run:
 
 ```yaml
 kubectl create namespace quick-start-application-ns
 ```
 
```

Comment:
the output of this command is
`namespace "quick-start-application-ns" created`
if we want to add that